agent/agent_endpoint_test.go

```diff
@@ -261,19 +264,16 @@ func TestAgent_Reload(t *testing.T) {
 		&structs.ServiceDefinition{Name: "redis-reloaded"},
 	}
 
-	ok, err := a.ReloadConfig(cfg2, cfg)
-	if err != nil {
+	if err := a.ReloadConfig(cfg2); err != nil {
 		t.Fatalf("got error %v want nil", err)
 	}
-	if !ok {
-		t.Fatalf("got ok %v want true")
-	}
 	if _, ok := a.state.services["redis-reloaded"]; !ok {
 		t.Fatalf("missing redis-reloaded service")
 	}
-	//verify that previous config's watch plans were stopped
-	for _, w := range cfg.WatchPlans {
-		if !w.IsStopped() {
+
+	// Verify that previous config's watch plans were stopped.
```

Comment:
Removed in https://github.com/hashicorp/consul/commit/4a3604a3eefbff4c7245d3790d26a37f5d5aa206.